packages/tasks/src/tasks/any-to-any/data.ts

```diff
@@ -16,16 +18,20 @@ const taskData: TaskDataCustom = {
 			id: "BAAI/Emu3-Gen",
 		},
 		{
-			description: "Janus is a versatile multimodal framework that decouples visual encoding for superior understanding and generation across tasks.",
+			description:
+				"Janus is a versatile multimodal framework that decouples visual encoding for superior understanding and generation across tasks.",
```

Comment:
is this a framework or a model? also we don't give any technical details, you can just say "Very powerful any-to-any model with xyz input and xyz output."